CMakeLists.txt

```diff
@@ -1985,16 +1985,16 @@ if(HID)
   if(HIDAPI_STATIC)
     message(STATUS "Linking internal libhidapi statically")
     add_library(mixxx-hidapi STATIC EXCLUDE_FROM_ALL)
-    target_include_directories(mixxx-hidapi SYSTEM PUBLIC lib/hidapi-0.8.0-rc1/hidapi)
+    target_include_directories(mixxx-hidapi SYSTEM PUBLIC lib/hidapi/hidapi)
```

Comment:
I removed the version number from the file path so we don't have to keep updating the build system when updating the library.